manifests/t/ThaUnknown/Miru/5.5.9/ThaUnknown.Miru.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created with WinGet Releaser v2 using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ThaUnknown.Miru
+PackageVersion: 5.5.9
+InstallerType: nullsoft
+Scope: user

```

```suggestion
Scope: machine
```

The installer appears to be `machine` scoped now. `Scope: user` was added in https://github.com/microsoft/winget-pkgs/pull/162720#discussion_r1676442652 back when Installer did install per-user